tests/kafkatest/services/verifiable_producer.py

```diff
@@ -79,8 +79,9 @@ def _worker(self, idx, node):
         node.account.ssh("mkdir -p %s" % VerifiableProducer.PERSISTENT_ROOT, allow_fail=False)
 
         # Create and upload log properties
+        producer_prop_file = self.prop_file
```

Comment:
Just to clarify - is `self.prop_file` necessary? One justification to keep it is that it provides a way to override configs.

But if the only way to pass in custom producer configs is to set or append to the `prop_file` field after constructing VerifiableProducer, that's a pretty confusing mechanism.